### PR TITLE
website: Add localisation to dashboard and tasks pages

### PR DESCRIPTION
**Pull request by rjmacarthy**
Hey, this PR includes changes to enable `i18n` support for the `dashboard` and the `task` configuration pages.

A keen eye would be appreciated when reviewing the changes to the `TaskTypes.ts` file in relation to the `tasks.json` file as strings were copied over and there were a fair few of them.

If you have any questions or concerns please let me know.

Thanks

### CODE REVIEW

**Review by fozziethebeat — APPROVED** [review]

**Comment by fozziethebeat** [comment]
Looks good but needs a small merge fix

File: website/src/components/Tasks/TaskTypes.tsx [hunk 1]
```diff
@@ -1,181 +1,161 @@
...
 
 export const TaskCategoryLabels: { [key in TaskCategory]: string } = {
-  [TaskCategory.Random]: "Grab a task!",
-  [TaskCategory.Create]: "Create",
-  [TaskCategory.Evaluate]: "Evaluate",
-  [TaskCategory.Label]: "Label",
+  [TaskCategory.Random]: "grab_a_task",
+  [TaskCategory.Create]: "create",
+  [TaskCategory.Evaluate]: "evaluate",
+  [TaskCategory.Label]: "label",
```
[the remark this answers is not in the document]
Comment (rjmacarthy) [reply website/src/components/Tasks/TaskTypes.tsx:42]: Thanks. I think this shows a title at the top of the page?  I simply replaced the strings with i18n keys.  I think refactoring this is out of scope for the pull request.

File: website/src/lib/i18n.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export const normalizei18nKey = (key: string) => key as unknown as TemplateStringsArray;
```
[the remark this answers is not in the document]
Comment (rjmacarthy) [reply website/src/lib/i18n.ts:1]: Thanks, what would you suggest as a name for this functions?

[the remark this answers is not in the document]
Comment (rjmacarthy) [reply website/src/lib/i18n.ts]: I have renamed to `getTypeSafei18nKey`, does it work?

File: website/src/components/Tasks/TaskTypes.tsx [hunk 3]
```diff
@@ -1,181 +1,161 @@
...
+  AssistantReply = "assistant_reply",
+  InitialPrompt = "initial_prompt",
+  LabelAssistantReply = "label_assistant_reply",
+  LabelInitialPrompt = "label_initial_prompt",
+  LabelPrompterReply = "label_prompter_reply",
+  PrompterReply = "prompter_reply",
+  Random = "random",
+  RankAssistantReplies = "rank_assistant_replies",
+  RankInitialPrompts = "rank_initial_prompts",
+  RankPrompterReplies = "rank_prompter_replies",
```
[the remark this answers is not in the document]
Comment (rjmacarthy) [reply website/src/components/Tasks/TaskTypes.tsx:25]: Thanks, yes I will update it.

**Review by AbdBarho — APPROVED** [review]

**Pull request merged**